website/config/routes.js

```diff
@@ -75,7 +75,14 @@ module.exports.routes = {
     }
   },// handles /docs and /docs/foo/bar
 
-  // 'GET /handbook/?*':  { skipAssets: false, action: 'handbook/view-basic-handbook' },// handles /handbook and /handbook/foo/bar
+  'GET /handbook/?*':  {
+    skipAssets: false,
+    action: 'handbook/view-basic-handbook',
+    locals: {
+      title: 'Handbook | Fleet for osquery',
+      description: 'View the Fleet handbook',
+    },
+  },// handles /handbook and /handbook/foo/bar
```

Comment:
double-check: do we also need to update the action that handles requests to sitemap.xml?